var/spack/repos/builtin/packages/graphite2/package.py

```diff
@@ -17,4 +17,6 @@ class Graphite2(CMakePackage):
 
     version('1.3.13', sha256='dd63e169b0d3cf954b397c122551ab9343e0696fb2045e1b326db0202d875f06')
 
+    depends_on('python@3.6:')
```

Comment:
Is this needed at run-time? Does the library need to be linked to?

Reply:
@darmac :point_up:

Reply:
It only need at run-time, I fixed that error in: https://github.com/spack/spack/pull/22126/commits/3bceb844b7ee416231347f856c92a9ba5d7eae2a